Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Image/StreamingImageAsset.h

```diff
@@ -144,6 +150,8 @@ namespace AZ
             //! mip chain index, not the level. And will fail for any level
             //! that resides in the tail mip chain.
             const ImageMipChainAsset* GetImageMipChainAsset(AZ::u32 mipLevel) const;
+
+            AZStd::vector<AZ::Name> m_tags;
```

Comment:
Why does the model asset uses unordered_set for its tags but not here?

Reply:
Also, can you add comment about why the asset may have more than one quality tags? such as an example use case.

Reply:
I changed to vectors instead of unordered_set, as there shouldn't be a lot of tags and they are iterated more often than inserted.

As for quality tags, we use them to set categories on our models/textures, such as "building", "tree", "background" and more. And depending on the quality settings it's interesting to be able to set multiple tags on assets